src/helper/Initializable.sol

```diff
@@ -0,0 +1,233 @@
+// Vendored from OpenZeppelin contracts with minor modifications:
```

Comment:
Even if PoC i think is easy you group all vendored contracts in some folder to make it easy to see what is the core contracts developed. Most contracts are vendored